templates/zypper.repo.j2

```diff
@@ -13,6 +13,7 @@ name=Datadog, Inc.
 enabled=1
 autorefresh=1
 baseurl={{ baseurl }}
+includepkgs={{ datadog_includepkgs }}
```

Comment:
I've tried testing this on OpenSUSE 42.3 and 15.2 and it works on neither, it seems that it just gets ignored. Does it work for you? Do you have any documentation reference that says it should work with zypper?

Reply:
You are indeed correct. My testing shows the same, includepkgs is ignored by Zypper.
I'll update the PR.